3rdParty/OUIDataset/create_oui_data.py

```diff
@@ -150,13 +157,12 @@ def main() -> None:
         if isinstance(line, bytes):
             line = line.decode("utf-8")
 
-        if line[0] in ["#", "\n"]:
+        if line.startswith(("//", "/*", " *", "*/", "\n", "\r\n", "static", "};")):
             continue
 
-        if line_elements := parse_mac_and_vendor(
-            [word for word in line.split("\t") if word.strip()]
-        ):
-            update_oui_dataset(oui_dataset, line_elements)
+        if line_elements := convert_line(line):
+            if line_elements := parse_mac_and_vendor(convert_line(line)):
```

Comment:
Something seems to be wrong here: you have nested `if` statements that use the same variable name (`line_elements`) so they override each other. Can you explain what these `if` statements are supposed to check?

Reply:
Actually, it was for preventing the error caused when `convert_line` return `None` but looks like I forgot to change variable of second `if` with `parse_mac_and_vendor(line_elements)`. But since first if successed it worked luckily so I missed it. The problem with `None` is there is no `len` of `None` so it raises an error in https://github.com/egecetin/PcapPlusPlus/blob/df622a8a1de05afda72d6849d6fd4607399cc22c/3rdParty/OUIDataset/create_oui_data.py#L77-L79

Maybe I can move this none check to inside of `parse_mac_and_vendor` it would be better for readability 🤔